src/lib.rs

```diff
@@ -60,6 +60,9 @@ pub mod binding;
 pub mod event;
 pub mod message;
 
+#[cfg(test)]
+pub mod test_data;
```

Comment:
It's a minor nit, but I don't like the name "test_data". I never like the word "data" to describe anything, but in this case, I prefer "fixtures" but I'd even settle for "util" over "data". 

Or maybe we rename "test_data" to "test" or "tests" or "testing" or "internal" or something else reasonable and then rename `data.rs` beneath that dir to `fixtures.rs`?

Reply:
Done ... renamed `test_data`->`test` and `data.rs`->`fixtures.rs`